pkg/volume/csi/csi_attacher_test.go

```diff
@@ -1627,7 +1627,11 @@ func newTestWatchPlugin(t *testing.T, fakeClient *fakeclient.Clientset) (*csiPlu
 	factory.Start(wait.NeverStop)
 	ctx, cancel := context.WithTimeout(context.Background(), TestInformerSyncTimeout)
 	defer cancel()
-	for ty, ok := range factory.WaitForCacheSync(ctx.Done()) {
+	syncedTypes := factory.WaitForCacheSync(ctx.Done())
```

Comment:
I don't see `.Informer` being called on the volume attachment informer. Is that intentional? Same question applies below.

Reply:
I didn't include this because we're now asserting the number of synced types, and it succeeds without having to add `.Informer`.

Reply:
Are you sure that one of those types is `VolumeAttachment`?

Reply:
Just checked this, one is for `CSIDriver` and one is for `VolumeAttachment`.

Reply:
As part of calling `Lister()`, `Informer()` gets called.